### ISSUE TITLE
New CARMA MA (1990_2000) case with added diagnostics and double radiation call

### ISSUE THREAD

**Issue by dan-visioni** [NCAR/CARMA_dev#30]
Case is here 
/glade/work/iquaglia/cases/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat.1990_2000

We have a double call to radiation (_d1) & added the diagnostics for the different contributions (with all the caveats in Simone's email).

**Comment by tilmes** [comment]
A few comments:
I would do a branch run using this REF_CASE (you need to add the name in env_run.xml) and use these restart files that you need to copy in your run directory:
/glade/scratch/tilmes/archive/FWmaCARMAnudged.f19_f19_mg17.carma_trop_strat.fixall.1987spinup/rest/1990-01-01

Also, we can also add AODVISdn etc. what I have seen from Francis' script.. but better check.

**Comment by ilaria-quaglia** [comment]
@tilmes, I submitted the simulation but got this error for which I can't find a solution:

**ERROR: ice_run_mct:: Internal sea-ice clock not in sync with Sync Clock**

Log file in /glade/scratch/iquaglia/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat.1990_2000/run/cesm.log.9728026.chadmin1.ib0.cheyenne.ucar.edu.230424-092823

**Comment by tilmes** [comment]
One quick comment first, all these variables are now available with "dn" so we can output those instead of the other once:
'ABSORBdn, 'AODVISdn', 'AODABSdn','AODUVstdn','AODNIRstdn','EXTINCTdn', 'EXTINCTUVdn', 'EXTINCTNIRdn'

regarding the problem, maybe you need to clean this up again and start over:
1. rm -rf /glade/scratch/iquaglia/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat.1990_2000/run/*
2. copy /glade/scratch/tilmes/archive/FWmaCARMAnudged.f19_f19_mg17.carma_trop_strat.fixall.1987spinup/rest/1990-01-01/* to /glade/scratch/iquaglia/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat.1990_2000/run/
3. in case_dir; ./case.setup --reset  , ./case.setup
4. ./case.build --clean , ./case.build
5. ./*.submit
If that does not work, we may have to try a hyrid run (but you would need to add the right ncdata file in your user_nl_cam file.

**Comment by ilaria-quaglia** [comment]
It didn't work. I can try with the hybrid run, but what is the ncdata file I need?  is /glade/p/acom/acom-climate/tilmes/inputdata/init/carma/FWmaCARMAnudged.f19_f19_mg17.carma_trop_strat.fixall.1987spinup.cam.i.1990-01-01-00000.nc? 

What is the difference between using the restart files in $case_dir/run and using the input file? Since for the hybrid run I have to specify RUN_REFCASE in env_run.xml, I can't understand which dataset is used to initialize the case.

**Comment by tilmes** [comment]
@ilaria-quaglia  Yes, this would be the right nc_data file:  /glade/p/acom/acom-climate/tilmes/inputdata/init/carma/FWmaCARMAnudged.f19_f19_mg17.carma_trop_strat.fixall.1987spinup.cam.i.1990-01-01-00000.nc
You will have to change that in user_nl_cam file

I am actually not sure why it does not work, but it may be because we changed the code.  A branch run, is a run that requires to have all the restart files in your run directory, and it should start exactly from the place the model stopped if we run from the same code base.  The branch run may not work when you have a different code base. So this may be an issue here.

The hybrid run takes the ocean and ice initial conditions but is not reproducing the run completely, and you can change the nc-data file (atmospheric initial conditions). This may however still not work, due to the new code base.

Therefore, I am thinking, we can just do a startup run, and make sure we have the right nc-data file. You can leave all the information you have in the env_run.xml file, just change the hybrid to startup.

Doing a startup in our case, will probably be fine, because we are prescribing SSTs and sea-ice anyways from file and so, that should not be a problem. If we are concerned about the land, we could also add a land initial condition file in user_nl_clm, but I don't think, this is needed.

**Comment by ilaria-quaglia** [comment]
This figure shows the total aerosol optical depth at 550 nm averaged  globally . 

Panel a is a comparison of the historical simulation between MAM4 (blue line, from /glade/campaign/acom/acom-climate/tilmes/CARMA/FWmaMAM4HIST.f19_f19_mg17.carma_trop_strat09.1980_2014), the previous run with CARMA using the code base #12 (green line) and the new CARMA run using the code base  in /glade/work/fvitt/camdev/carma_trop_strat (yellow line). 

Panel b is a zoom on 1991  to highlight the differences: both CARMA runs have lower values than MAM4 and differences between the two CARMA runs are smaller than 0.02 (or 0.007 if the first month of simulation is excluded).

Panel c shows  in CARMA each contribution to AODVISdn.

Panel d shows the total absorption at 500 nm between the two CARMA simulations, and panel e the absorption by black carbon at the same wavelength for the new CARMA simulation.

![aod_gm](https://user-images.githubusercontent.com/125593798/235777542-5e376bbf-935e-4fd1-a89a-a51aa95f6290.png)

**Comment by ilaria-quaglia** [comment]
here we have the stratospheric aerosol optical depth averaged between 80°S-80°N at three wavelengths for the three runs compared with GloSSAC. First row is for UV, second row for VIS and third row for NIR. The legend specifies the wavelength and whether the AOD is day only or day night (**I forgot to save AODVISstdn for the new CARMA run**). The second column is a zoom on 1991.

New CARMA run and MAM in the stratosphere look comparable for now (in background conditions) and differences between the two CARMA runs are of the same order of magnitude of the differences we saw for the total AOD. I don't know what is different between the code base since I missed the last two meetings.




![aodst_gm](https://user-images.githubusercontent.com/125593798/235940326-b32b0881-0d17-4737-9c1e-a4ffe4de5e54.png)

**Comment by ilaria-quaglia** [comment]
The the global burdens in CARMA are weird. They are mostly the same curve but not exactly the same (see second figure).
![burden](https://user-images.githubusercontent.com/125593798/235944011-3b1f4403-0e82-4892-b6c7-5b6be3edc0cb.png)
![burden_carma](https://user-images.githubusercontent.com/125593798/235946608-21ab8eb7-057d-4601-a244-0ebb392f9d87.png)

**Comment by ilaria-quaglia** [comment]
New case using the codebase #14 and adding new diagnostics (mass mixing ratio) in:
/glade/work/iquaglia/cases/FWmaCARMAHIST.f19_f19_mg17.carma_trop_strat14.1990_2000

<VAR>_calc means that the sum over the bins for each VAR is calculated offline. Differences are much smaller than 1%.
@fvitt 
![mmr_cnf](https://github.com/NCAR/CARMA_dev/assets/125593798/74e898a2-3918-4b2d-9dc1-f5c8ea897e93)
